components/micro-gateway-core/src/main/ballerina/gateway/filters/authn_filter.bal

```diff
@@ -24,18 +24,14 @@ import ballerina/time;
 import ballerina/io;
 import ballerina/reflect;
 
-
 // Authentication filter
 
 @Description {value:"Representation of the Authentication filter"}
 @Field {value:"filterRequest: request filter method which attempts to authenticated the request"}
 public type AuthnFilter object {
 
     public OAuthnAuthenticator oauthnHandler;// Handles the oauth2 authentication;
-    public http:AuthnHandlerChain authnHandlerChain;
-
-
-    public new (oauthnHandler, authnHandlerChain) {}
+    public boolean oauth2Enabled = false;
```

Comment:
Shall we rename the variable name to "isOAuth2Enabled" as it is a boolean variable?